README.md

```diff
@@ -1,3 +1,43 @@
-# Membership
+# Node.js Foundation - Membership
 
-Repo to work out the system for individual membership in the foundation.
+## What types of members are there?
+
+* Enterprise
+* Individual
+* Student
+* Active Contributor
+
+## What are the conditions of eligibility?
+
+TBD - Join the discussion on this at [issues/8](https://github.com/nodejs/membership/issues/8)
+
+## How to join
```

Comment:
As I mentioned [here](https://github.com/nodejs/membership/issues/6#issuecomment-107222887), I believe the GH issues are only for running for the board seat, not for joining the foundation. That should be done via a form on the website, and a paypal link, I would assume.